packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1427,6 +1433,18 @@ export class ContainerRuntime
 			summaryFormatVersion: metadata?.summaryFormatVersion,
 			disableIsolatedChannels: metadata?.disableIsolatedChannels,
 			gcVersion: metadata?.gcFeature,
+			options: JSON.stringify(runtimeOptions),
+			featureGates: JSON.stringify({
+				compressionDisabled,
+				opReentryDisabled,
+				chunkingDisabled,
+				attachReorderDisabled: this.mc.config.getBoolean(
+					"Fluid.ContainerRuntime.disableAttachOpReorder",
+				),
+				partialFlushDisabled: this.mc.config.getBoolean(
+					"Fluid.ContainerRuntime.DisablePartialFlush",
+				),
+			}),
 		});
 
 		ReportOpPerfTelemetry(this.context.clientId, this.deltaManager, this.logger);
```

Comment:
changes in this file seems unrelated to the core of PR, I'd suggest pushing them separately.